task/build-image-index/0.1/build-image-index.yaml

```diff
@@ -147,3 +150,59 @@ spec:
       capabilities:
         add:
           - SETFCAP
+
+  - image: quay.io/redhat-appstudio/sbom-utility-scripts-image:d4935b35eb0849841412d5d1e481a812e939f2bf
+    name: create-sbom
+    computeResources:
+      limits:
+        memory: 512Mi
+        cpu: 200m
+      requests:
+        memory: 256Mi
+        cpu: 100m
+    env:
+    - name: IMAGE_URL
+      value: $(results.IMAGE_URL.path)
+    - name: IMAGE_DIGEST
+      value: $(results.IMAGE_DIGEST.path)
+    script: |
+      INDEX_IMAGE_PULLSPEC="${IMAGE_URL}@${IMAGE_DIGEST}"
```

Comment:
Have you tested this? I think this concatenates the result paths, not the result values